ml-agents/mlagents/trainers/tests/test_rl_trainer.py

```diff
@@ -92,3 +93,12 @@ def test_rl_trainer(add_policy_outputs, add_rewards_outputs, num_vis_obs):
     for rewards in trainer.collected_rewards.values():
         for agent_id in rewards:
             assert rewards[agent_id] == 0
+
+
+def test_clear_update_buffer():
+    trainer = create_rl_trainer()
+    trainer.training_buffer = construct_fake_buffer()
+    trainer.training_buffer.append_update_buffer(2, batch_size=None, training_length=2)
+    trainer.clear_update_buffer()
+    for _, arr in trainer.training_buffer.update_buffer.items():
```

Comment:
No is_empty() method on the buffer? :(

Reply:
No... maybe we should add it. Also I use this in the buffer: `len(next(iter(self.update_buffer.values())))` to get its length - there probably should be a length method as well.

Reply:
Your call - I'm OK with this as it is, but I think that would be a little nicer.

Reply:
I'll make a separate PR, unless we decide to refactor the buffer anyways